main/rabbit/definitions.json

```diff
@@ -0,0 +1,277 @@
+{
+  "rabbit_version" : "3.6.10",
+  "users" : [
+	{
+	  "name" : "guest",
+	  "password_hash" : "m8PefsW4FcXeHN8M4B8OCAxHdao=",
+	  "hashing_algorithm" : "rabbit_password_hashing_md5",
+	  "tags" : "administrator"
+	}
+  ],
+  "vhosts" : [
+	{
+	  "name" : "/"
+	},
+	{
+	  "name" : "/lidskasila"
+	}
+  ],
+  "permissions" : [
+	{
+	  "user" : "guest",
+	  "vhost" : "/lidskasila",
+	  "configure" : ".*",
+	  "write" : ".*",
+	  "read" : ".*"
+	},
+	{
+	  "user" : "guest",
+	  "vhost" : "/",
+	  "configure" : ".*",
+	  "write" : ".*",
+	  "read" : ".*"
+	}
+  ],
+  "parameters" : [ ],
+  "global_parameters" : [
+	{
+	  "name" : "cluster_name",
+	  "value" : "rabbit@ef4b8b34ac2c"
+	}
+  ],
+  "policies" : [ ],
+  "queues" : [
```

Comment:
Ten command je pomalý a musel bych ho pouštět po každém `docker-compose up`, tohle se naimportuje během chvilky. Jestli to někdo bude používat a narazí na novou frontu, tak PR sem udělá během chvilky.